libsignal-service/src/push_service.rs

```diff
@@ -67,8 +67,26 @@ pub const STICKER_PATH: &str = "stickers/%s/full/%d";
 pub const KEEPALIVE_TIMEOUT_SECONDS: Duration = Duration::from_secs(55);
 pub const DEFAULT_DEVICE_ID: u32 = 1;
 
+#[derive(Debug)]
+pub enum ServiceIdType {
+    // Account Identity
+    ACI,
+    // Phone number identity
+    PNI,
+}
```

Comment:
```suggestion
#[derive(Debug)]
pub enum ServiceIdType {
    // Account Identity
    Aci,
    // Phone number identity
    Pni,
}
```

That's more rusty...